hardhat.config.ts

```diff
@@ -13,6 +13,7 @@ const config: HardhatUserConfig = {
         url: String(process.env.ETH_RPC_URL),
         blockNumber: Number(process.env.FORK_BLOCK),
       },
+      loggingEnabled: true,
```

Comment:
Does this have any significant impact on performance? From my quick test it seems to make no noticeable difference, but just want to confirm

Reply:
I doubt it's relevant, yes. But turning it off shouldn't hurt either.